diff --git a/node_modules/switch-joy-con/index.js b/node_modules/switch-joy-con/index.js
index 91c762e..7c3753d 100644
--- a/node_modules/switch-joy-con/index.js
+++ b/node_modules/switch-joy-con/index.js
@@ -5,13 +5,23 @@ const JoyConRight = require("./joy-con-right");
 function listConnectedJoyCons() {
   const devices = HID.devices();
   return devices
-    .filter(device => device.product.match(/Joy-Con/))
+    .filter(device => {
+      return (
+        device.hasOwnProperty("product") &&
+        (device.product.match(/Joy-Con/) ||
+          (device.manufacturer.match("Nintendo") &&
+            device.product.match("Wireless Gamepad")))
+      );
+    })
     .map(device => {
       return Object.assign({}, device, {
         open() {
-          if (device.product.match(/\(L\)/)) {
+          if (device.product.match(/\(L\)/) || device.productId === 8198) {
             return new JoyConLeft(device.path);
-          } else if (device.product.match(/\(R\)/)) {
+          } else if (
+            device.product.match(/\(R\)/) ||
+            device.productId === 8199
+          ) {
             return new JoyConRight(device.path);
           }
         }
